### PR TITLE
Add documentation for Kura Inspector

### PR DESCRIPTION
**Pull request by outoftardis**
### Description of the Change

Add docs for Kura Inspector.

image

### CODE REVIEW

**Comment by appetrosyan** [comment]
A few general comments. 
1. This PR is made to the release branch. We're trying to keep the release branches as stable as possible and that includes documentation. 
2. This doc is added to the `docs/` folder. Instead it should be a standalone `README.md` in the `tools/kura_inspector` directory.

File: docs/source/tools/kura-inspector.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+Kura Inspector is a CLI tool to inspect blocks in disk storage ([Kura](../iroha_2_whitepaper.html#data-storage)).
+
+With Kura Inspector you can inspect the disk storage regardless of the operating status of Iroha and print out block contents in a human-readabe format.
+
+## Examples
+
+- Print the contents of the latest block:
+
+      kura_inspector print
```
Comment (appetrosyan) [inline docs/source/tools/kura-inspector.md:11]: We prefer the triple backtick syntax. Indentation-based blocks often break.

File: docs/source/tools/kura-inspector.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+Run Kura Inspector:
+
+    kura_inspector [OPTIONS] <SUBCOMMAND>
+
+### Options
+
+|     Option     |                     Description                     |  Default value   |  Type   |
+| -------------- | --------------------------------------------------- | ---------------- | ------- |
+| `-f`, `--from` | The block height from which to start the inspection | The latest block | `usize` |
```
Comment (appetrosyan) [inline docs/source/tools/kura-inspector.md:31]:
```suggestion
| `-f`, `--from` | block height of start of range for inspection | current block height | Positive integer |
```
`usize` is a rust-specific term. While most programmers will know what it means, we want to give as much information as possible. 

Block height and block number are not the same. Block 0 corresponds to block height of 1.

File: docs/source/tools/kura-inspector.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+### Errors
+
+An error in Kura Inspector occurs if one the following happens:
+
+- `kura_inspector` fails to configure `kura::BlockStore` ([Kura Configurations](../references/config.html#kura))
+- `kura_inspector` [fails](#print-errors) to run the `print` subcommand
+
+## `print`
+
+The `print` command reads data from the `block_store` and prints the results to `output`.
```
Comment (appetrosyan) [inline docs/source/tools/kura-inspector.md:49]:
```suggestion
The `print` command reads data from the `block_store` and prints the results to the specified `output`.
```

File: docs/source/tools/kura-inspector.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+|      Option      |                                      Description                                      |  Default value  |  Type   |
+| ---------------- | ------------------------------------------------------------------------------------- | --------------- | ------- |
+| `-n`, `--length` | The number of blocks to print. The excess is truncated.                               | 1               | `usize` |
+| `-o`, `--output` | Where to write the results of the inspection: valid data and [errors](#print-errors). | Standard Output |         |
```
Comment (appetrosyan) [inline docs/source/tools/kura-inspector.md:54]:
```suggestion
| `-o`, `--output` | Where to write the results of the inspection: valid data and [errors](#print-errors). | Standard Output |          file |
```

On Unix almost everything that isn't a string or a number, is a file. You could also have written `/dev/stdout`, which is the standard convention in [man pages](https://en.wikipedia.org/wiki/Man_page)

File: docs/source/tools/kura-inspector.md (hunk 2 above)
Comment (outoftardis) [reply docs/source/tools/kura-inspector.md]: You use `usize` in `docs` to describe configs, which is why I used the same term her. 
Updated this description.

**Comment by outoftardis** [comment]
> 1. This PR is made to the release branch. We're trying to keep the release branches as stable as possible and that includes documentation.

Ok. I've used this branch as a target because this was the branch I was originally pointed to. Should I change the target branch to `iroha2-dev`?

> 2. This doc is added to the `docs/` folder. Instead it should be a standalone `README.md` in the `tools/kura_inspector` directory.

Changed the location

**Pull request closed without merging**